bats/fb-katello-client.bats

```diff
@@ -83,30 +91,50 @@ load fixtures/content
 }
 
 @test "install katello-agent" {
+  if tIsVersionNewer "${KATELLO_VERSION}" 4.10; then
+    skip "katello-agent is no longer supported as of Katello 4.10"
+  fi
   tPackageInstall katello-agent && tPackageExists katello-agent
 }
 
 @test "30 sec of sleep for groggy gofers" {
+  if tIsVersionNewer "${KATELLO_VERSION}" 4.10; then
+    skip "katello-agent is no longer supported as of Katello 4.10"
+  fi
   sleep 30
 }
 
-@test "install package remotely (katello-agent)" {
+@test "install package remotely" {
   run yum -y remove gorilla
-  timeout 300 hammer host package install --host "${HOSTNAME}" --packages gorilla
+  if tIsVersionNewer "${KATELLO_VERSION}" 4.10; then
+    timeout 300 hammer job-invocation create --feature katello_package_install --inputs 'package=gorilla' --search-query "name = ${HOSTNAME}"
+  else
+    timeout 300 hammer host package install --host "${HOSTNAME}" --packages gorilla
```

Comment:
I must admit I always found it weird that the REX-based package install is not something like
```
hammer host package install --host "${HOSTNAME}" --packages gorilla --provider rex
```

Reply:
Why does a user need to care about provider?

Reply:
I see it as creating a common invocation method no matter the operation. We can have a special invocation method just for package operations, where do we draw the line between special commands and general use of the REX hammer interface?

I am not saying I necessarily like the current REX invocation as it's a bit... wordy. And likely based off the structure of the API more than thinking about the users point of view.

Reply:
> Why does a user need to care about provider?

Exactly.

In the UI, there is a default one and the user can select an alternative one if they want to with a single click.

With hammer, it's totally different depending on the provider.

Reply:
I definitely agree that it's a bit less convenient than the old katello agent commands. With katello agent gone, we could consider cleaning up the hammer commands to make them more convenient.
@jeremylenz think it's worth an addition to the tasks relating to post katello-agent removal cleanup?

Reply:
> cleaning up the hammer commands to make them more convenient.

What specifically do you have in mind? Keeping the old `hammer host package` commands and redirecting them to REX? Or improving the `hammer job invocation create` commands?

Reply:
See https://github.com/Katello/hammer-cli-katello/pull/904

Reply:
My approach is to just warn the user and ask them to use the job-invocations
I do not want to be in the business of redirecting because it would require us to keep hammer commands consistent with the job template changes.